tools/features/com.liferay.ide.hidden/feature.xml

```diff
@@ -158,7 +158,7 @@
          id="org.objectweb.asm"
          download-size="0"
          install-size="0"
-         version="0.0.0"
+         version="5.2.0.v20170126-0011"
```

Comment:
I don't think we need to specify this one.  We should keep the versions removed.